airflow/api_fastapi/core_api/routes/ui/config.py

```diff
@@ -61,6 +61,10 @@ def get_configs() -> ConfigResponse:
         "audit_view_included_events": conf.get("webserver", "audit_view_included_events", fallback=""),
         "audit_view_excluded_events": conf.get("webserver", "audit_view_excluded_events", fallback=""),
         "test_connection": conf.get("core", "test_connection", fallback="Disabled"),
+        "color_log_error_keywords": conf.get("logging", "color_log_error_keywords", fallback="").split(","),
```

Comment:
1. there's a `conf.getlist` function
2. This is a UI setting, not a logging setting
3. Now we have structured logging (where the log level comes form the "source" directly without needing to pase , I don't think we should be looking for keywords.
4. I really really really REALLY don't want us to add more config options!

Also "Task succeded without error" would get highlighted as an error. Very prone to false positives.

Reply:
This is not a new configuration. This was already present in Airflow 2.10. This is a port of the implementation from the old UI to the new UI. We have use cases where we show logs from external systems like Spark, HDFS etc that are not structured from the triggerer as a log group like "::group::stdout". The keywords help in highlighting the relevant line and improves debugging.

> Also "Task succeded without error" would get highlighted as an error. Very prone to false positives.

Agreed, this is like a simple `ctrl+f` in the browser and has been more useful than a hindrance,

Config docs : https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#color-log-error-keywords
Airflow 2.10 implementation : https://github.com/apache/airflow/issues/37443

Reply:
3.0 is the opportunity to decide which config to remove too.

Reply:
I am mostly with Ash. I had mostly false-positives with the highlighting so far which ended-up that we disabled all keywords from default as users called because being confused.

For Error/Warning I think the highlighting should be really oriented from the log sub system and log level as we have structured data.

I'd be OK to have the highlighting with keyword, if not "Exception", "Error" and "Warning" are used per default in config (as this would be over-lapping with the log level). It might be use-ful for cases where custom words (other than error...) are to be highlighted.